src/lib/y2storage/proposal_settings.rb

```diff
@@ -414,6 +414,7 @@ def load_features
       load_feature(:proposal, :other_delete_mode)
       load_feature(:proposal, :delete_resize_configurable)
       load_feature(:proposal, :lvm_vg_strategy)
+      load_feature(:proposal, :lvm_vg_reuse)
```

Comment:
This `load_feature` is for reading values from the YaST control file. It makes no sense to add `lv_vg_reuse` if we are not adding it to the list of settings that can be configured through the control file.

Reply:
i know ... i just took the opportunity when already touching it. It might happen that we will want to defer the behavior in the future products

Reply:
So you propose to document the new value in the control file documentation and all that. Isn't it?

Because adding support in the code without making it fully official (mentioning it in the feature, adding it to the control file config, etc.) doesn't seem the way to go.

Reply:
Do not continue with that idea